src/server/services/collection.service.ts

```diff
@@ -592,6 +594,16 @@ export const saveItemInCollections = async ({
       );
   }
 
+  // Check for updates to featured models
+  if (input.modelId && collections.some((c) => c.userId === -1 && c.name === 'Featured Models')) {
+    await bustFeaturedModelsCache();
+    const versions = await dbWrite.modelVersion.findMany({
```

Comment:
Should we use `dbRead` in here instead?
```suggestion
    const versions = await dbRead.modelVersion.findMany({
```